canonical/vanilla-framework
# Percy examples combination - `patterns/search-box`

Deletes old `default-dark` example and sets up a redirect to `default?theme=dark`.
Because there is only one search box example variant, there is no need for a `combined.html` for it currently.

## QA
- Verify https://vanilla-framework-5169.demos.haus/docs/examples/patterns/search-box/default-dark redirects to https://vanilla-framework-5169.demos.haus/docs/examples/patterns/search-box/default?theme=dark
